ui/perfherder/alerts/AlertTableRow.jsx

```diff
@@ -45,14 +51,22 @@ export default class AlertTableRow extends React.Component {
     this.state = {
       starred: this.props.alert.starred,
       checkboxSelected: false,
+      showSideBySideLink: false,
       icons: [],
     };
   }
 
   componentDidMount() {
     const { alert } = this.props;
+    const showSideBySideLink =
```

Comment:
I believe this logic can be moved in `render()`. I don't see why `showSideBySideLink` should be kept in state.